easybuild/tools/robot.py

```diff
@@ -356,7 +356,7 @@ def resolve_dependencies(easyconfigs, modtool, retain_all_deps=False):
 
 def search_easyconfigs(query, short=False, filename_only=False, terse=False):
     """Search for easyconfigs, if a query is provided."""
-    search_path = build_option('robot_path')
+    search_path = build_option('search_paths')
```

Comment:
@ocaisa This is not what I had in mind, I was thinking about letting `--search-paths` empty by default, and doing this:

```python
search_path = build_option('robot_path')
if not search_path:
    search_path = [os.getcwd()]
extra_search_paths = build_option('search_paths')
if extra_search_paths:
    search_path.extend(extra_search_paths)